assembl/static2/js/app/components/administration/voteSession/pageForm.jsx

```diff
@@ -0,0 +1,154 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import { FormGroup } from 'react-bootstrap';
+import SectionTitle from '../sectionTitle';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+import FileUploader from '../../common/fileUploader';
+import TitleWithHelper from '../../common/titleWithHelper';
+import { getEntryValueForLocale } from '../../../utils/i18n';
+import {
+  updateVoteSessionPageTitle,
+  updateVoteSessionPageSubtitle,
+  updateVoteSessionHeaderImage,
+  updateVoteSessionPageInstructionsTitle,
+  updateVoteSessionPageInstructionsContent,
+  updateVoteSessionPagePropositionsTitle
+} from '../../../actions/adminActions/voteSession';
+
+type PageFormProps = {
+  headerTitle: string,
+  headerSubtitle: string,
+  headerImgUrl: string,
+  headerImgMimeType: string,
+  instructionsTitle: string,
+  instructionsContent: string,
+  propositionSectionTitle: string,
+  selectedLocale: string,
+  handleHeaderTitleChange: Function,
+  handleHeaderSubtitleChange: Function,
+  handleHeaderImageChange: Function,
+  handleInstructionsTitleChange: Function,
+  handleInstructionsContentChange: Function,
+  handlePropositionSectionTitleChange: Function
```

Comment:
there is probably something to do for the types of the dispatched redux actions, we should look at it some day